tests/uefistored/conftest.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+@pytest.fixture(scope='module')
+def imported_sb_vm(imported_vm):
```

Comment:
I've tried the sb test on a non UEFI VM, and it fails most tests (which looks good), but I think the tests should be completely skipped when the VM is not in UEFI mode.

Reply:
Some tests pass, such as `test_boot_success_when_launching_db_signed_images`